content/content-tool/tool/src/webapp/vm/content/sakai_resources_list.vm

```diff
@@ -8,6 +8,9 @@
     vertical-align: middle !important;
     text-align:left !important;
   }
+  .table td{
+    text-indent: 0em !important;
+  }
```

Comment:
did you intend for this change to be in this pull request?

Reply:
No, that's part of SAK-50368.